From c053d30e41147735663e2c2082eccb6243340fcb Mon Sep 17 00:00:00 2001
From: Grace Cham <hscham@chromium.org>
Date: Mon, 12 Dec 2022 15:54:19 +0900
Subject: [PATCH] Select sha1_nacl.cc implementation.

Select sha1_nacl.cc, md5_nacl.cc implementation.
NACL doesn't have boringssl either, we can just select NACL implementation for
MD5/SHA1 used in upstream Chromium.

Change-Id: I82e14ea2a001fdb435da9aa813379191aec8d4f4
patch-name: long-term-0600-select-nacl-md5-sha1-implementation.patch
---
 base/hash/md5.h  | 2 +-
 base/hash/sha1.h | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/base/hash/md5.h b/base/hash/md5.h
index aa889f350e..fab9794d42 100644
--- a/base/hash/md5.h
+++ b/base/hash/md5.h
@@ -11,7 +11,7 @@
 #include "base/strings/string_piece.h"
 #include "build/build_config.h"
 
-#if BUILDFLAG(IS_NACL)
+#if BUILDFLAG(IS_NACL) || 1
 #include "base/hash/md5_nacl.h"
 #else
 #include "base/hash/md5_boringssl.h"
diff --git a/base/hash/sha1.h b/base/hash/sha1.h
index 29626e5853..c357078fb8 100644
--- a/base/hash/sha1.h
+++ b/base/hash/sha1.h
@@ -14,7 +14,7 @@
 #include "base/containers/span.h"
 #include "base/strings/string_piece.h"
 #include "build/build_config.h"
-#if BUILDFLAG(IS_NACL)
+#if BUILDFLAG(IS_NACL) || 1
 #include "base/hash/sha1_nacl.h"
 #else
 #include "base/hash/sha1_boringssl.h"
-- 
2.42.0.582.g8ccd20d70d-goog

